src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -331,8 +333,8 @@ public void cannotCreateRequestForUnknownItem() {
       .withRequesterId(patronId));
 
     assertThat(postResponse, hasStatus(HTTP_UNPROCESSABLE_ENTITY));
-    assertThat(postResponse.getJson(), hasErrorWith(allOf(
-      hasMessage("Item does not exist"))));
+    assertThat(postResponse.getJson(), hasErrorWith(
+      hasMessage("Item does not exist")));
```

Comment:
Can this now fit on one line (less than 100 characters)?

```suggestion 
    assertThat(postResponse.getJson(), hasErrorWith(hasMessage("Item does not exist")));
```

Reply:
Yes. Fixed.